documentation/src/docs/metrics.md

```diff
@@ -209,39 +255,48 @@ public class PravegaRequestProcessor extends FailingRequestProcessor implements
     …
 }
 ```
-From the above example, we can see the required steps to register and use a metric in desired class and method:
+Using the above example, following are the required steps to register and use a metric in the desired class and method:
+
+1. Get a `StatsLogger` from `MetricsProvider`.
 
-1. Get a StatsLogger from MetricsProvider: 
 ```
 StatsLogger STATS_LOGGER = MetricsProvider.getStatsLogger();
 ```
-2. Register all the desired metrics through StatsLogger:
+2. Register all the desired metrics through `StatsLogger`.
 ```
 static final OpStatsLogger CREATE_STREAM_SEGMENT = STATS_LOGGER.createStats(SEGMENT_CREATE_LATENCY);
 ```
-3. Use these metrics within code at appropriate place where you would like to collect and record the values.
+3. Use these metrics within code at the appropriate places where you would like to collect and record the values.
+
 ```
 Metrics.CREATE_STREAM_SEGMENT.reportSuccessEvent(timer.getElapsed());
 ```
-Here CREATE_STREAM_SEGMENT is the name of this metric, and CREATE_STREAM_SEGMENT is the name of our Metrics logger, it will track operations of createSegment, and we will get the time of each createSegment operation happened, how long each operation takes, and other numbers computed based on them.
+The `CREATE_STREAM_SEGMENT` is the name of this metric, and `CREATE_STREAM_SEGMENT` is the name of our Metrics logger. It will track operations of `createSegment`, and we will get the time (i.e. time taken by each operation and other numbers computed based on them) for each `createSegment` operation happened.
+
+### Output Example of OpStatsLogger
+
+An example output of `OpStatsLogger CREATE_SEGMENT` reported through [CSV](https://metrics.dropwizard.io/3.1.0/manual/core/#csv) reporter.
 
-### 2.1.1 Output example of OpStatsLogger 
-An example output of OpStatsLogger CREATE_SEGMENT reported through CSV reporter:
 ```
-$ cat CREATE_STREAM_SEGMENT.csv 
+$ cat CREATE_STREAM_SEGMENT.csv
 t,count,max,mean,min,stddev,p50,p75,p95,p98,p99,p999,mean_rate,m1_rate,m5_rate,m15_rate,rate_unit,duration_unit
 1480928806,1,8.973952,8.973952,8.973952,0.000000,8.973952,8.973952,8.973952,8.973952,8.973952,8.973952,0.036761,0.143306,0.187101,0.195605,calls/second,millisecond
 ```
 
 
-### 2.2. Example for Dynamic Gauge and OpStatsLogger(Histogram) 
-This is an example from io.pravega.controller.store.stream.AbstractStreamMetadataStore. In this class, we report a Dynamic Gauge which represents the open transactions and  one histogram (CREATE_STREAM).
+## Example for Dynamic Gauge and OpStatsLogger(Histogram)
+
+This is an example from `io.pravega.controller.store.stream.AbstractStreamMetadataStore`. In the class, `AbstractStreamMetadataStore` we report a Dynamic Gauge which represents:
+
+- Open Transactions
+- One Histogram (`CREATE_STREAM`).
+
 ```java
 public abstract class AbstractStreamMetadataStore implements StreamMetadataStore  {
```

Comment:
Another copy-pasted code where most of the code is irrelevant to this document

Reply:
Hyperlink provided